test/e2e/scaffold/ingress.go

```diff
@@ -276,10 +276,7 @@ spec:
             - --apisix-route-version
             - %s
             - --watch-endpointslices
-          volumeMounts:
-           - name: webhook-certs
-             mountPath: /etc/webhook/certs
-             readOnly: true
+          %s
       volumes:
        - name: webhook-certs
```

Comment:
Should this also be set  only when enableWebhook=true ？

Reply:
It doesn't seem to affect